test/end-to-end/test_studio_can_build_packages.ps1

```diff
@@ -4,6 +4,14 @@
 # test, and leave more detailed testing to the build output to e2e tests for hab-plan-build
 hab origin key generate $env:HAB_ORIGIN
 
+function Convert-LastBuildEnvToPs1 {
```

Comment:
Could we move this to `run_e2e_test_core.ps1`? And use the function [here](https://github.com/habitat-sh/habitat/blob/c378874ecae4cc4dd750fb10cbf6cbe302f9e1a2/.expeditor/scripts/end_to_end/run_e2e_test_core.ps1#L248) as well.

Reply:
good call. Done @davidMcneil